kasa/tests/test_device_factory.py

```diff
@@ -1,16 +1,25 @@
-# type: ignore
+"""Module for testing device factory.
```

Comment:
Changes in this file belong into the tests polish PR.

Reply:
As above `device_factory` tests break due to this PR.